hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -75,6 +77,7 @@
   private final long lockAcquireTimeout;
 
   private FileIO fileIO;
+  private Method alterTableMethod;
```

Comment:
Since the version of Hive is unlikely to change, can we make this a `static` field?